### PR TITLE
Fix queries against information_schema with uppercase names

### PR DESCRIPTION
**Pull request by kokosing**
Fix queries against information_schema with uppercase names

### CODE REVIEW

**Comment by kokosing** [comment]
Fixes #11344

File: presto-main/src/main/java/com/facebook/presto/connector/informationSchema/InformationSchemaMetadata.java [hunk 1]
```diff
@@ -249,6 +250,7 @@ private boolean isTablesEnumeratingTable(SchemaTableName schemaTableName)
         Optional<Set<String>> schemas = filterString(constraint, SCHEMA_COLUMN_HANDLE);
         if (schemas.isPresent()) {
             return schemas.get().stream()
+                    .map(schema -> schema.toLowerCase(ENGLISH))
```
Comment (sopel39) [inline presto-main/src/main/java/com/facebook/presto/connector/informationSchema/InformationSchemaMetadata.java]: just filter name that contains upercase?

File: presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java [hunk 2]
```diff
@@ -4836,6 +4836,14 @@ public void testInformationSchemaFiltering()
                 "SELECT 'customer' table_name");
     }
 
+    @Test
+    public void testInformationSchemaUppercaseName()
+    {
+        assertQuery(
+                "SELECT table_name FROM information_schema.tables WHERE table_catalog = 'LOCAL' AND table_schema = 'TINY' AND table_name = 'ORDERS'",
```
Comment (electrum) [inline presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java]: Should we do three separate queries, one for each filter?

**Comment by kokosing** [comment]
Rebased and applied comments

**Pull request merged**